Server/Plugins/APIDump/Classes/BlockArea.lua

```diff
@@ -1621,7 +1621,7 @@ return
 						Type = "number",
 					},
 				},
-				Notes = "Sets the block meta at the specified absolute coords",
+				Notes = "Sets the block meta at the specified absolute coords. Any call to SetBlockMeta will not generate a simulator update (water, lava, redstone), consider using SetBlock instead.",
```

Comment:
Similarly for cBlockArea, it doesn't do any simulator updates at all (for speed when setting a large area).